lib/components/regions_map/interactor/view_models/regions_map_bloc.dart

```diff
@@ -48,7 +48,16 @@ class RegionsMapBloc extends Bloc<RegionsMapEvent, RegionsMapState> {
     } else {
       final result = await GetUserLocationUseCase().run();
       if (result.isError) {
-        emit(state.copyWith(pageState: PageState.failure));
+        // User deny location permissions set default location: Mexico city coords
+        // and hide current location button.
+        emit(state.copyWith(newPlace: state.newPlace.copyWith(lng: -99.085092, lat: 19.461416)));
```

Comment:
instead of using numbers directly, change it to be a constant 

mexicoLocation = lng: -99.085092, lat: 19.461416

Reply:
Moved to a const